src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -80,7 +80,7 @@ public void c024928()
         }
 
         [WorkItem(2771, "https://github.com/dotnet/roslyn/issues/2771")]
-        [Fact]
+        [Fact(Skip = "This test is flaky under low memory conditions; it fails in StringBuilder.ToString in the parser")]
```

Comment:
Please move this change into a separate commit, this is a second PR that contains this change.